### PR TITLE
Add a GitHub Action to notify campaigns team if someone mentions campaigns

### PR DESCRIPTION
**Pull request by mrnugget**
I'm not sure if this works?

### CODE REVIEW

File: .github/workflows/campaigns-notify.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Notify the campaigns team if campaigns are mentioned in the issue
```
Comment (eseliger) [inline .github/workflows/campaigns-notify.yml:1]: Maybe a little long for the actions UI, but not a blocker
```suggestion
# Notify the campaigns team if campaigns are mentioned in the issue
name: notify-campaigns
```

File: .github/workflows/campaigns-notify.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+jobs:
+  create_comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-ecosystem/action-create-comment@v1
+        if: contains(github.event.comment.body, 'campaigns')
+        with:
+          github_token: ${{ secrets.github_token }}
```
Comment (eseliger) [inline .github/workflows/campaigns-notify.yml:14]: is this actually lower-case? or is it case insensitive?

**Review by eseliger — APPROVED** [review]
Nice job, cool approach of combining existing pieces :)

File: .github/workflows/campaigns-notify.yml (hunk 2 above)
Comment (mrnugget) [reply .github/workflows/campaigns-notify.yml:14]: In other actions files it's lower case. But I'm actually not sure. I have no clue how to "try" this 😬

Comment (eseliger) [reply .github/workflows/campaigns-notify.yml:14]: :shipit: and mention us 😬

Comment (eseliger) [reply .github/workflows/campaigns-notify.yml:14]: Oh one thing that just came to my mind: would we also want to skip if the team/campaigns label is already applied? 
(Ie it's created by us)

**Review by LawnGnome — APPROVED** [review]
Approved, mostly because I think we're just going to need to try this, per the inline conversation.

File: .github/workflows/campaigns-notify.yml (hunk 2 above)
Comment (mrnugget) [reply .github/workflows/campaigns-notify.yml:14]: Yeah, that's what I wanted to add next. I wrote this up between meetings 🙈 I'll try this tomorrow!

**Pull request merged**